book/version_control.md

```diff
@@ -220,23 +220,18 @@ git clone https://github.com/<user>/<repo>.git --branch=v0.1.0
 
 
 (excluding-from-git)=
-### Excluding information from Git repositories
+## Avoid commiting sensitive information to Git repositories
```

Comment:
Isn't this duplicating something that has been covered already ? Feels a bit fragmented